platform/virt/Makefile

```diff
@@ -208,6 +208,7 @@ BOOT_OBJDIR=    $(OBJDIR)/boot
 UEFI_SRCS= \
 	$(SRCDIR)/aarch64/elf64.c \
 	$(SRCDIR)/aarch64/uefi.c \
+	$(SRCDIR)/boot/elf.c \
```

Comment:
This file shouldn't actually be needed here, but in SRCS-uefi in platform/pc/boot/Makefile instead: the uefi_kernel_complete() function is only used if uefi_arch_options->load_to_physical is false, which is the case for the x86 UEFI loader, but not for the aarch64 one.